src/main/java/com/minecolonies/coremod/blocks/BlockPaperwall.java

```diff
@@ -74,6 +77,8 @@ public MapColor getMapColor(final IBlockState state)
     /**
      * Convert the given metadata into a BlockState for this Block
      */
+    @NotNull
+    @Deprecated
     @Override
     public IBlockState getStateFromMeta(final int meta)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)